src/annotator/config/index.js

```diff
@@ -159,8 +162,9 @@ export function getConfig(appContext = 'annotator', window_ = window) {
   let filteredKeys = configurationKeys(appContext);
   filteredKeys.forEach(name => {
     const configDef = configDefinitions[name];
-    // Get the value from the configuration source
-    config[name] = configDef.getValue(settings);
+    const coerceFn = configDef.coerce ? configDef.coerce : name => name; // use no-op if omitted
+    // Get the value from the configuration source and run through an optional coerce method
+    config[name] = coerceFn(configDef.getValue(settings));
```

Comment:
I think I would probably have written this without the no-op function as:

```js
const value = configDef.getValue(settings);
config[name] = configDef.coerce ? configDef.coerce(value) : value;
```

I find this very slightly easier to follow.